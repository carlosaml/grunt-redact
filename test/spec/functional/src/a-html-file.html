<html>
<body>
<h1>Hello there</h1>
<div feature="facebookLogin">Facebook login is ON</div>
<div feature="googleLogin">Google login is ON</div>
<div not-feature="googleLogin">Google login is OFF</div>
</body>
</html>