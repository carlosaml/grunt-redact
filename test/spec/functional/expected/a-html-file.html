<html>
<body>
<h1>Hello there</h1>
<div>Facebook login is ON</div>

<div>Google login is OFF</div>
</body>
</html>